Casamari/MNC001.xml

```diff
@@ -303,32 +641,93 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                        <date notBefore="1500" notAfter="1699">16th-17th century</date>
-                        <desc>
-                           Fine and careful handwriting.
-                        </desc>
+                        <date notBefore="1600" notAfter="1799">17th/18th century writing</date>
+                        <seg type="script">
+                           Small and slightly rushed handwriting.
+                           Another hand, featured by a smaller and less trained writing, intervenes on <locus from="47vb" to="48rb"/>, perhaps in order to fill a textual lacuna.
+                           One more change of hand might have occurred towards the end, see e.g. <locus target="#80v"/>, with characters written more closely (cp. <ref target="#h3"/>) and in a pale black ink.
+                        </seg>
+                        <seg type="rubrication">
+                           Groups of two lines on the incipit pages alternating with black lines, e.g. on <locus target="#2r #36r #50r"/>; 
+                           titles of the subvidisions of the works, indicated with the words <foreign xml:lang="gez">ክፍል፡</foreign> or <foreign xml:lang="gez">ምዕራፍ፡</foreign>;
+                           the title of <ref target="#ms_i1"/> in the upper margin of <locus target="#2r"/>;
+                           the name of <persName ref="PRS12013Ablanewos"/>, possibly a later owner, written on erasure in the subscription on <locus target="#936rc"/>;
+                           elements of the punctuations signs, including the text dividers, and of the Ethiopic numerals.
+                        </seg>
+                        <list type="abbreviations">
+                           <item>
+                              <abbr>ምዕ፡</abbr> or <abbr>ምዕራ፡</abbr> for <expan>ምዕራፍ፡</expan> (e.g. <locus target="#39va #67ra #68rb"/>)
+                           </item> 
+                           <item>
+                              <abbr>ክፍ፡</abbr> for <expan>ክፍል፡</expan> (e.g. <locus target="#53rc #54ra"/>)
+                           </item> 
+                        </list>
+                     </handNote>
+                     <handNote xml:id="h2" script="Ethiopic">
+                        <locus from="81ra" to="128rc"/>
+                        <seg type="ink">Black, red</seg>
+                        <date notBefore="1600" notAfter="1799">17th/18th century writing</date>
+                        <seg type="script">
+                          Small handwriting. It becomes rapid and increasingly irregular towards the end. The characters are boradly spaced, e.g. on <locus target="#128r"/>.
+                        </seg>
                         <seg type="rubrication">
+                           One or two lines on the incipit pages alternating with black lines, e.g. on <locus target="#81r #97vc #125vb"/>; 
+                           sometimes one single line of the incipits, e.g. on <locus target="#116ra"/>.
```

Comment:
; instead of a dot at the end of the line?
if with dot, then the next word capitalized **T**itles

Reply:
I meant the semicolon ;